libutils/definitions.h

```diff
@@ -25,6 +25,14 @@
 #ifndef CFENGINE_DEFINITIONS_H
 #define CFENGINE_DEFINITIONS_H
 
+/*****************************************************************************
+ * Metric prefixes
+ *****************************************************************************/
+#define KIBIBYTE(n) (n * 1024)
+#define MEBIBYTE(n) (n * 1024 * 1024)
+#define GIBIBYTE(n) (n * 1024 * 1024 * 1024)
+#define TEBIBYTE(n) (n * 1024 * 1024 * 1024 * 1024)
```

Comment:
```suggestion
#define KIBIBYTE(n) (n * 1024L)
#define MEBIBYTE(n) (n * 1024L * 1024L)
#define GIBIBYTE(n) (n * 1024L * 1024L * 1024L)
#define TEBIBYTE(n) (n * 1024L * 1024L * 1024L * 1024L)
```

I guess you have to write it like this to avoide overflow.